src/BuilderMethodExtension.php

```diff
@@ -51,7 +51,13 @@ public function setBroker(Broker $broker)
      */
     public function hasMethod(ClassReflection $classReflection, string $methodName): bool
     {
-        if ($classReflection->isSubclassOf(Model::class) && !isset($this->methods[$classReflection->getName()])) {
+        if (!isset($this->methods[$classReflection->getName()]) && (
+                $classReflection->isSubclassOf(Model::class)
+                || preg_match(
+                    '/@mixin\s+' . preg_quote('\\' . Builder::class) . '/',
```

Comment:
Should we make the leading slash optional?

Reply:
@twistor That `/` is the pattern delimiter!

Reply:
No, the `\\` for the namepsace.

Reply:
Oh, the backslashes!

Reply:
@twistor sorry for delay, but I have a lot of work now, but promise to you to resolve issue in next few days

Reply:
@twistor I've moved mixin detection to a separate class and it will return class names without the leading slash, but if you'll try to use such form, for example, in IDE it will show incorrect namespace error (checked in PHPStorm). Of course, in IDE can be used even short class names, because it can parse use section, but in this case it is imposible to detect class namespace from doc block only.